system/web/context/Response.cfc

```diff
@@ -260,7 +260,13 @@ component accessors="true" {
 		numeric totalRecords = 0,
 		numeric totalPages   = 1
 	){
-		variables.pagination = arguments;
+		variables.pagination = {
```

Comment:
We can't do this @homestar9 because if you do that, then the user CAN'T pass in extra arguments to include into the pagination structure.  So if we wanted to do this, then we would still have to incorporate the origination structure into the internal storage to allow for extra attributes.

Reply:
@gpickin thoughts?

Reply:
Gotcha. That makes sense.

Side note:
I've always wondered why there's a `pagination` struct in the response object anyway.  It's always smelled funny to me since it only applies when returning paginated data.  Before I was using the default Coldbox response object, I was sending paginated data as a `Page` object mementified in the `data` property like this:
```
{
	"data": {
		"pageNumber": 1,
		"maxRows": 10,
		"totalRows": 100,
		"data": [
			... // the actual paged data
		]
	},
	"error": false,
	"messages": []
}
```